### PR TITLE
Fixes min_learning_rate bug

### LINKED ISSUE #145: min_learning_rate in ReduceLROnPlateau scheduler
My labmate was using the `ReduceOnPlateau` scheduler and gets an error related tot he arg `min_learning_rate`. 

This is because we inherit from the [pytorch scheduler](https://pytorch.org/docs/1.13/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html#torch.optim.lr_scheduler.ReduceLROnPlateau), which uses `min_lr`. It looks like we reverted this a couple months ago and changed `min_lr` to `minimum_lr` [here](https://github.com/CUNY-CL/yoyodyne/blob/master/yoyodyne/schedulers.py#L151). 

Reverting it back should fix the bug.

### PR DESCRIPTION
**Pull request by Adamits**
Closes #145

### CODE REVIEW

**Review by kylebgorman — APPROVED** [review]
LGTM.
